packages/turf-center-median/index.js

```diff
@@ -0,0 +1,71 @@
+import centerMean from '@turf/center-mean';
+import distance from '@turf/distance';
+import { point, isObject, isNumber } from '@turf/helpers';
+import { featureEach } from '@turf/meta';
+import { getCoord } from '@turf/invariant';
+
+/**
+ * Takes a {@link FeatureCollection} of points and calculates the median center,
+ * algorithimically. It uses the algorithm described in:
+ *
+ * Harold W. Kuhn and Robert E. Kuenne, “An Efficient Algorithm for the
+ * Numerical Solution of the Generalized Weber Problem in Spatial
+ * Economics,” _Journal of Regional Science_ 4, no. 2 (1962): 21–33,
+ * doi:{@link https://doi.org/10.1111/j.1467-9787.1962.tb00902.x}.
+ *
+ * James E. Burt, Gerald M. Barber, and David L. Rigby, _Elementary
+ * Statistics for Geographers_, 3rd ed., New York: The Guilford
+ * Press, 2009, 150–151.
+ *
+ * @name centerMedian
+ * @param {FeatureCollection<Point>} features GeoJSON Feature or Geometry
```

Comment:
Might want to call the parameter `points` instead of `features`, that way it's explicitly indicating that you need to provide Points.

Also I would replace the `GeoJSON Feature or Geometry` as the description